src/main/pages/setup-docker/docker-cli.adoc

```diff
@@ -152,127 +226,126 @@ These variables can be set in your local environment shell before running and th
+|`--offline` |Loads Docker images from the `backup/*.tar` directory during a pre-boot mode of the CLI. Used if you are performing an installation or start while disconnected from the Internet.
 |===
 
-You can reinstall Che on a folder that is already initialized and preserve your `che.env` values by passing the `--reinit` flag.
-
-'''''
-
-_offline_
-
-Saves all of the Docker images that Che requires into `/backup/*.tar` files. Each image is saved as its own file. If the `backup` folder is available on a machine that is disconnected from the Internet and you start Che with `--offline`, the CLI pre-boot sequence will load all of the Docker images in the `/backup/` folder.
+You can reinstall Che on a directory that is already initialized and preserve your `che.env` values by passing the `--reinit` flag.
 
-`--list` option will list all of the core images and optional stack images that can be downloaded. The core system images and the CLI will always be saved, if an existing TAR file is not found. `--image:<image-name>` will download a single stack image and can be used multiple times on the command line. You can use `--all-stacks` or `--no-stacks` to download all or none of the optional stack images.
+*_offline_*
 
-'''''
+The `offline` command saves all the Docker images that Che requires in the `/backup/*.tar` files. Each image is saved as its own file. If the `backup` directory is available on a machine that is disconnected from the Internet and you start Che with the `--offline` command option, the CLI pre-boot sequence loads all the Docker images in the `/backup/` directory.
 
-_restart_
+The `--list` option lists all the core images and optional stack images that can be downloaded. The core system images and the CLI are always saved if an existing `TAR` file is not found. The `--image:<image-name>` command option downloads a single-stack image and can be used multiple times on the command line. You can use the `--all-stacks` option or the `--no-stacks` option to download all or none of the optional stack images.
 
-Performs a `stop` followed by a `start`, respecting `--pull`, `--force`, `--offline`, `--skip:config`, `--skip:preflight`, and `--skip:postflight`.
+*_restart_
 
-'''''
+The `restart` command performs a `stop` action followed by a `start` action, respecting the `--pull`, `--force`, `--offline`, `--skip:config`, `--skip:preflight`, and `--skip:postflight` command options.
 
-_restore_
+*_restore_*
 
-Restores `/instance` to its previous state. You do not need to worry about having the right Docker images. The normal start / stop / restart cycle ensures that the proper Docker images are available or downloaded, if not found.
+The `restore` command restores the `/instance` directory to its previous state. The start/stop/restart cycle ensures that the proper Docker images are available or downloaded, if not found.
```

Comment:
Replace the slashes with "the start, stop, and restart cycle". The comma before "if not found" is not necessary.